### ISSUE TITLE
accessibility for linked cms-images

### ISSUE THREAD

**Issue by Liverson** [shopware/shopware#10846]
### Technical TODO

When linking a image in cms elements actually it is not possible to add a title to the link and make it accessible.

Also for youtube integration, the iframe has no title

**Comment by ravenokko** [comment]
Hi @Liverson and thanks for submitting! 

For image links, it is acceptable to [use the image alt text to describe the link purpose](https://www.w3.org/WAI/WCAG22/Techniques/html/H30#tests). In 6.7 we also offer the ability to add a Link description, which is then included in the anchor as an aria-label (see: [passed example 4](https://www.w3.org/WAI/standards-guidelines/act/rules/c487ae/)) 

<img width="908" height="670" alt="Image" src="https://github.com/user-attachments/assets/d707bcbd-3f52-4441-bd80-1c89d09d30ef" />

The youtube iframe title should have been fixed with https://github.com/shopware/shopware/issues/11291

Is there a particular scenario we're missing? Let me know if you're still having issues

**Comment by ravenokko** [comment]
closing this issue as it should already have been resolved, but please reopen this if you're still having problems!
